python/ray/ml/checkpoint.py

```diff
@@ -1,35 +1,126 @@
-import abc
-from typing import Dict, Optional
+import shutil
+import tarfile
+import tempfile
+
+import os
+from typing import Optional
 
 import ray
+from ray import cloudpickle as pickle
```

Comment:
I'd do `as cpickle` to make it clear this is cloudpickle

Reply:
We use the same line everywhere else in the Ray repo, so let's keep it consistent here